lib/shaman/tryout_apps.rb

```diff
@@ -0,0 +1,7 @@
+module Shaman
+  module TryoutApps
+    DEVELOPMENT_BASE_URI = 'http://localhost:3000'
+    STAGING_BASE_URI = 'https://staging-infinum.tryoutapps.com'
```

Comment:
We support other tenants (e.g. Dept) on Tryoutapps so it would be great to support a user provided option (and defaulting to Infinum)

Reply:
I'm not sure what exactly is the scope of changes you are describing. I think it's best to start with a new task in the Productive.

Reply:
Nevermind, I see we still kept the `SERVER_URL` ENV logic from before so nothing will change for Dept 👍